src/column-layout/interfaces.ts

```diff
@@ -19,6 +21,8 @@ export interface ColumnLayoutProps extends BaseComponentProps {
 
   /**
    * Controls whether dividers are placed between rows and columns.
+   *
+   * Note: This is not supported when used with `minColumnWidth`.
```

Comment:
Not for this PR, but a general suggestion – how about marking this as `@deprecated` too?